wallet/src/params.rs

```diff
@@ -76,4 +77,17 @@ impl NodeParams {
             }
         }
     }
+
+    /// Get the address of an existing JsonRpcClient actor.
+    pub fn get_client(&self) -> NodeClient {
+        let lock = self.client.clone();
```

Comment:
Will this clone the `NodeClient` or it will only increase the reference count and return an `Arc`? To avoid that question you can use `Arc::clone` explicitly, or annotate the type as `let lock: Arc<_> = ...`

```suggestion
        let lock = Arc::clone(&self.client);
```

Reply:
This clones the `Arc` indeed, but that fact is concealed by the magic dereference of the `Arc` when it is used afterwards. Great suggestion — the most explicit the code, the better.